classes/submission/form/SubmissionSubmitStep1Form.inc.php

```diff
@@ -38,6 +38,20 @@ function fetch($request) {
 		$sectionOptions = array('0' => '') + $sectionDao->getTitles($this->context->getId(), !$canSubmitAll);
 		$templateMgr = TemplateManager::getManager($request);
 		$templateMgr->assign('sectionOptions', $sectionOptions);
+		$templateMgr->assign('sectionId', $request->getUserVar('sectionId'));
+
+		// Get section policies for this context
+		$sectionPolicies = array();
+		foreach ($sectionOptions as $sectionId => $sectionTitle) {
+			$section = $sectionDao->getById($sectionId);
+
+			$sectionPolicy = $section ? $section->getLocalizedPolicy() : null;
+			$sectionPolicyPlainText = trim(PKPString::html2text($sectionPolicy));
+			if (strlen($sectionPolicyPlainText) > 0)
```

Comment:
(See https://github.com/pkp/pkp-lib/pull/4135/files#r224624538)

Reply:
See my comments in the linked PR